lib/tasks/serve.js

```diff
@@ -172,6 +180,7 @@ exports.shutdown = function (mage, options, cb) {
 	}
 
 	async.series([
+		teardownModules,
 		closeServers,
```

Comment:
`closeServers` would close both mmrp and http. I would need to split that phase into two, and teardown in between; is that OK?